src/org/zaproxy/zap/extension/pscanrulesAlpha/files/licenses/xmpcore-license.txt

```diff
@@ -0,0 +1,38 @@
+The XMP Core library is distribued under terms of a modified BSD
+license.  The license is re-published in full below.
```

Comment:
How about "reproduced" instead of "re-published"? The latter doesn't sound right.